mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
+ignore_missing_imports = True
+strict_optional = False
+allow_redefinition = True
+show_error_context = False
+show_column_numbers = True
```

Comment:
github is complaining about a lack of newline at the end of this file I think

Reply:
Fixed!